docs/4.0/docs/examples/use-private-registry.md

```diff
@@ -0,0 +1,69 @@
+# how to use sealos private registry

```

Comment:
```suggestion
# How to use sealos private registry
```

Please use upper case letters properly (e.g. First letter of sentences, topics, etc.)